src/views/index.html

```diff
@@ -108,7 +108,7 @@ <h5>
                             <div class="two half unit row">
                                 <div class="unit column grows">
                                     <h5>
-                                        Applications for Hack Cambridge Ternary are now closed.<br><span class="secondary">Sign up for info about HC 2019!</span>
+                                        Applications for Hack Cambridge 2019 will open later this year in the autumn.<br><span class="secondary">Sign up to be kept in the loop!</span>
```

Comment:
Probably worth removing the "Sign up below to be notified when applications open for Hack Cambridge 2019." text underneath as it's basically a duplication of the secondary heading.